File: apps/sdkwork-rtc-flutter-mobile/lib/bootstrap/app_services.dart
import 'package:sdkwork_rtc_flutter_mobile_rtc/sdkwork_rtc_flutter_mobile_rtc.dart';

import 'app_auth.dart';
import 'sdk_clients.dart';

RtcAppServices createAppServices({RtcAppSession? session}) {
  final activeSession = session ?? loadAppSession();
  if (activeSession == null) {
    throw StateError(
      'RTC app services require an authenticated session; sign in before creating services.',
    );
  }
  final clients = createSdkClients(session: activeSession);
  return createRtcAppServices(clients.app.appSdk);
}

File: apps/sdkwork-rtc-h5/src/bootstrap/appServices.ts
import { createRtcAppServices, type RtcAppServices } from "@sdkwork/rtc-h5-rtc";

import { getAppSdkClient } from "./appClient";

export function createAppServices(): RtcAppServices {
  return createRtcAppServices(getAppSdkClient());
}
